config/certmanager/certificate-metrics.yaml

```diff
@@ -0,0 +1,20 @@
+# The following manifests contain a self-signed issuer CR and a metrics certificate CR.
+# More document can be found at https://docs.cert-manager.io
+apiVersion: cert-manager.io/v1
+kind: Certificate
```

Comment:
How does those certificate works then ? Are they loaded into k8s ?

Reply:
That's the part i'm trying to figure out

Reply:
Because the chart do not deploy without the cert :)